src/app/app.component.ts

```diff
@@ -9,6 +10,7 @@ import { views } from '../constants';
 export class AppComponent {
   username: string = 'Alyx Vance';
```

Comment:
generally, type annotation is redundant in this case. So, I'd suggest removing it when a value is set during a property declaration.